Add missing int types to enable builds in strict(er) C99 mode.

diff --git a/jam.c b/jam.c
index 4bc1b90ab48d821d..d3c0c83cae05c4cc 100644
--- a/jam.c
+++ b/jam.c
@@ -166,6 +166,7 @@ extern char **environ;
 # endif
 # endif
 
+int
 main( int argc, char **argv, char **arg_environ )
 {
 	int		n;
diff --git a/mkjambase.c b/mkjambase.c
index 26224c977d206cf5..bcad9af1a5ebc866 100644
--- a/mkjambase.c
+++ b/mkjambase.c
@@ -24,6 +24,7 @@
 # include <stdio.h>
 # include <string.h>
 
+int
 main( int argc, char **argv, char **envp )
 {
 	char buf[ 1024 ];
